redbot/cogs/warnings/warnings.py

```diff
@@ -76,6 +84,49 @@ def __init__(self, bot: Red):
         else:
             await ctx.send(_("Custom reasons have been disabled."))
 
+    @warningset.command()
+    @commands.guild_only()
+    async def toggledm(self, ctx: commands.Context):
+        """Toggle warns being sent to the users DM's"""
+        guild = ctx.guild
+        toggle = await self.config.guild(guild).toggle_dm()
+        await self.config.guild(guild).toggle_dm.set(not toggle)
+        toggle = await self.config.guild(ctx.guild).toggle_dm()
```

Comment:
You have the toggle variable already, so you should probably use `not toggle` instead of making another Config call.